geant4.spec

```diff
@@ -1,10 +1,13 @@
-### RPM external geant4 11.1.1
+### RPM external geant4 11.1.2
 ## INCLUDE compilation_flags
 ## INCLUDE compilation_flags_lto
 %define use_vecgeom 0
-%define tag 35f304cde19a3e2dfe3bb941d4fc76c2fe45fa26
-%define branch cms/v%{realversion}
-%define github_user cms-externals
+#%define tag 35f304cde19a3e2dfe3bb941d4fc76c2fe45fa26
+#%define branch cms/v%{realversion}
+#%define github_user cms-externals
+%define tag cf4c6712c60cdf2b4d8799e34e176917c2de358d
+%define branch v%{realversion}
```

Comment:
@iarspider , there is no `v%{realversion}` branch in official geant4 repo,  I would suggest to just use `master`